tests/oss-fuzz/README.md

```diff
@@ -0,0 +1,80 @@
+# Fuzzing AVIF on oss-fuzz
+
+*Last updated: 2024-01-22* 
+*Authors: [Vincent Rabaud](mailto:vrabaud@google.com)*
+*Status: **Current***
+
+# Overview
+
+This document provides links and knowledge about fuzzing libavif on oss-fuzz.
+
+# Creating targets
+
+[fuzztest](https://github.com/google/fuzztest/) is the framework of choice. Any fuzztest test declared in libavif/tests/CMakeLists.txt will be picked up and added to the list.
+
+# Links
+
+The main page [https://oss-fuzz.com](https://oss-fuzz.com)[/](https://oss-fuzz.com/) links to:
+
+- the [crashes](https://oss-fuzz.com/testcases?project=libavif&open=yes) : some tests do not have a bug created. They are flaky, duplicates. Still, it is worth going over the list in case there is a flaky true positive.
```

Comment:
this page shows a login screen, do you know what the ACLs are? can any github user access them?